rdopkg_tar/tar_changes.py

```diff
@@ -152,9 +153,14 @@ def upload_source(osdist, tarball, args_new_sources):
         clear_old_changes_sources()
         run(cmd, 'upload', tarball, direct=True)
 
-def main():
+
+def parse_args(parsed_params):
+    """
+    parse args from the spec file and assign to the collection parsed_params
+    args assigned: branch, patches_branch, changes, tarball, osdist, args
+    """
```

Comment:
I'd rather initialize parsed_params in the parse_args function, we can just return it